gems/activerecord/6.1/activerecord-6.1.rbs

```diff
@@ -6,6 +6,10 @@ module ActiveRecord
     extend DelegatedType
   end
 
+  module Core
+    def values_at: (*Symbol | String) -> Array[untyped]
+  end
```

Comment:
Could you make copies of this signature to other versions?
It would be nice if we could provide this type for Rails-7.0, 7.1 and 7.2.

Reply:
@tk0miya 

I copied this signature to other versions without 6.0.
Thanks!